### PR TITLE
Add Kinesis module

### PR DESCRIPTION
**Pull request by simonrondelez**
https://github.com/skyscrapers/engineering/issues/206

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+variable "name" {
+  description = "Name of the stack"
+}
+
+variable "environment" {
+  description = "Environment to deploy on"
+}
+
+variable "tags" {
+  description = "(Optional, Default: []) List of map of additional tags"
```
Comment (venturel) [inline variables.tf]: I'd remove (Optional, Default: [])

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,13 @@
+resource "aws_kinesis_stream" "kinesis_stream" {
+  name             = "${var.project}-${var.name}-${var.environment}"
+  shard_count      = "${var.kinesis_chard_count}"
+  retention_period = "${var.kinesis_retention_period}"
+
+  tags = "${var.tags}"
```
Comment (venturel) [inline main.tf:6]: Shouldn't we add some default tag?

Comment (simonrondelez) [reply main.tf:6]: I don't see anything specific to add. Any proposal?

Comment (simonrondelez) [reply main.tf:6]: by doing it like this we allow the user to set its own tags to the resource

Comment (venturel) [reply main.tf:6]: Nope it was just an idea but I am not sure what default tag can be added

**Review by venturel — APPROVED** [review]

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,13 @@
+resource "aws_kinesis_stream" "kinesis_stream" {
+  name             = "${var.project}-${var.name}-${var.environment}"
+  shard_count      = "${var.kinesis_chard_count}"
```
Comment (duboisph) [inline main.tf]: Typo: chard

File: variables.tf [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  description = "Environment to deploy on"
+}
+
+variable "tags" {
+  description = "List of map of additional tags"
+  type        = "list"
+  default     = []
+}
+
+variable "kinesis_chard_count" {
```
Comment (duboisph) [inline variables.tf]: Typo

**Review by duboisph — CHANGES_REQUESTED** [review]
Just a typo. But can you also make sure there are proper descriptions for the inputs (even if it is obvious ;) )?

**Review by duboisph — APPROVED** [review]

**Pull request merged**